src/helm/proxy/retry.py

```diff
@@ -41,7 +41,8 @@ def wait(attempts: int, delay: float) -> float:
         Wait function to pass into `Retrying` that logs and returns the amount of time to sleep
         depending on the number of attempts and delay (in milliseconds).
         """
-        next_delay = _retrying.exponential_sleep(attempts, delay)
+        del delay  # unused
```

Comment:
Wait, why do we need to do this?

Reply:
It's a Python convention to delete unused arguments to document that are not used and to prevent them from accidentally being used ([reference](https://stackoverflow.com/a/14836005)).